query/autocomplete.js

```diff
@@ -59,23 +61,28 @@ query.filter( peliasQuery.view.boundary_rect );
 **/
 function generateQuery( clean ){
 
-  var vs = new peliasQuery.Vars( defaults );
+  const vs = new peliasQuery.Vars( defaults );
+
+  let logStr = '[query:autocomplete] [parser:addressit] ';
```

Comment:
It can also be a `const` because Javascript.  Apparently things can be appended to a const.